Tests/Functionnal/src/Acme/AppBundle/Tests/Controller/JediControllerTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Acme\AppBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class JediControllerTest extends WebTestCase
```

Comment:
We should either make this test work or delete this file and make functionnal tests later